packages/components/src/components/data-table/data-table.jsx

```diff
@@ -12,10 +13,35 @@ import ThemedScrollbars from '../themed-scrollbars';
 */
 
 class DataTable extends React.PureComponent {
+    cache = React.createRef();
```

Comment:
Let's rename this variable to a more descriptive name as it may cause confusion for others, we can choose a different name such as `cell_measurer_cache_ref`, `cache_ref`.

Reply:
got it, I will.